python/grizzly/grizzly/dataframeweld.py

```diff
@@ -1,13 +1,13 @@
 import numpy as np
 import pandas as pd
 
-import grizzly_impl
-from lazy_op import LazyOpResult, to_weld_type
+from grizzly import grizzly_impl
```

Comment:
This does not work with python 2 for some reason. I am not really sure as to how to proceed with being able to support both python 2 and python 3 without putting it into a grizzly_impl class.

Reply:
Have you tried adding grizzly_impl import in __init__ and then here doing `from . import grizzly_impl`?

I'd still recommend using `from .grizzly_impl import <functions>` and only relative imports within the module.

Reply:
ah thanks! the `from . import grizzly_impl` worked